lesson04/script.js

```diff
@@ -1,11 +1,11 @@
 const ul = document.getElementById('js-list');
-const listContents = [
+const lists = [
  {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
  {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
 ]
 
 const fragment = document.createDocumentFragment();
-for(const list of listContents){
+for(const list of lists){
     const li = document.createElement('li');
     const a = document.createElement('a');
     const img = document.createElement('img');
```

Comment:
Change listContents to lists.
Because to make the object name easy to understand.